_auto/latest.py

```diff
@@ -40,7 +40,7 @@ def key(n):
 This is important to avoid edge cases like a 4.10.x release being marked under the 4.1 release cycle.
 """
 def releases_matches(r, prefix):
-  return (r.startswith(prefix) and (r == prefix or r.startswith(prefix + '.')))
+  return (r.startswith(prefix) and (r == prefix or r.startswith(prefix + '.') or r.startswith(prefix + '-')))
```

Comment:
can we get an extra parantesis to make this much more understandable
i mean x and y or z is a bit suspecios because is it `( x and y ) or z` or  `x and (y  or z)` 
@captn3m0

Reply:
opss it was my mistake ignore what i have write above